examples/tts/tacotron2.py

```diff
@@ -245,6 +253,7 @@ def create_eval_dags(
             tb_writer_func=partial(tacotron2_eval_log_to_tb_func, tag=tagname),
             eval_step=eval_freq,
             tb_writer=neural_factory.tb_writer,
+            eval_at_start=not do_not_eval_at_start,
```

Comment:
ehehehe 👍

Reply:
I switched from train loss to eval loss for tts test.